test/util/client.go

```diff
@@ -1,13 +1,23 @@
 package util
 
 import (
+	"os"
+	"path"
 	"path/filepath"
 
+	kclient "k8s.io/kubernetes/pkg/client"
+
 	"github.com/openshift/origin/pkg/client"
 	configapi "github.com/openshift/origin/pkg/cmd/server/api"
-	kclient "k8s.io/kubernetes/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/tokencmd"
 )
 
+// GetBaseDir returns the base directory used for test.
+func GetBaseDir() string {
+	return cmdutil.Env("BASETMPDIR", path.Join(os.TempDir(), "openshift-"+Namespace()))
```

Comment:
I don't see where this is used in extended

On Thu, Sep 24, 2015 at 4:27 AM, Michal Fojtik <notifications@github.com>
wrote:

> In test/util/client.go
> <https://github.com/openshift/origin/pull/4771#discussion_r40295585>:
>
> >  )
> >
> > +// GetBaseDir returns the base directory used for test.
> > +func GetBaseDir() string {
> > +	return cmdutil.Env("BASETMPDIR", path.Join(os.TempDir(), "openshift-"+Namespace()))
>
> I think for extended we have /tmp/openshift-extended... in case you want
> to run integration and extended in parallel.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4771/files#r40295585>.
>